charts/cost-view/hack/verify_queries.py

```
@@ -0,0 +1,71 @@
+for doc in docs:
+    name = doc["metadata"]["name"]
+    spec = doc["spec"]
+    print(f"--- {name}")
+    mem = sqlite3.connect(":memory:")
+
+    for qname, q in (spec.get("queries") or {}).items():
+        sql = q.get("sql", {}).get("query")
+        if not sql:
+            print(f"  skip {qname} (not a sql query)"); continue
+        wrapped = f"SELECT coalesce(json_agg(t), '[]'::json) FROM ({subst(sql).rstrip().rstrip(';')}) t"
+        r = subprocess.run(["psql", DB, "-tAX", "-v", "ON_ERROR_STOP=1", "-c", wrapped],
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)